src/Traisi.Questions/src/travel-diary-logger/travel-diary-logger.component.ts

```diff
@@ -210,8 +225,18 @@ export class TravelDiaryLoggerComponent extends SurveyQuestion<ResponseTypes.Str
 
 	//Select a date from the calendar
 	public updateTripDate() {
+		
 		let coordinates: any[] = this.getCords(this.selectedDate);
-		this.generateMap(coordinates);
+		if(coordinates.length <= 0)
+		{
+			document.getElementById("map").innerHTML = "";
```

Comment:
As a style issue, we should stick to using single quotes in .ts files for strings